src/containers/shared/components/Transaction/PaymentChannelClaim/Description.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { findNode, normalizeAmount } from '../../../transactionUtils'
+import { Account } from '../../Account'
+import { TransactionDescriptionProps } from '../types'
+import { PaymentChannelClaim } from './types'
+import { useLanguage } from '../../../hooks'
+
+export const Description = ({
+  data,
+}: TransactionDescriptionProps<PaymentChannelClaim>) => {
+  const { t } = useTranslation()
+  const language = useLanguage()
+  const deleted = findNode(data, 'DeletedNode', 'PayChannel')
+  const modified = findNode(data, 'ModifiedNode', 'PayChannel')
+  const node = deleted || modified
+  const change =
+    node && node.PreviousFields && node.PreviousFields.Balance
+      ? node.FinalFields.Balance - node.PreviousFields.Balance
+      : null
+
+  return (
+    <>
+      <div data-test="account-line">
+        {t('transaction_initiated_by')} <Account account={data.tx.Account} />
+      </div>
+      <div data-test="channel-line">
+        {t('update_payment_channel')}{' '}
+        <span className="channel">{data.tx.Channel}</span>
+      </div>
+      {data.tx.Balance && (
+        <div key="line3" data-test="balance-line">
```

Comment:
Since there's no longer line 1 and line 2, should this key be changed to "balance-line"? (I spent a bit searching for line1/line2 so it doesn't seem very descriptive anymore)

Reply:
the keys were there for when it returned an array which react needs keys.